libhb/reader.c

```diff
@@ -315,7 +315,7 @@ static int reader_init( hb_work_object_t * w, hb_job_t * job )
     // fifos that will be needed (+1 for null terminator)
     r->fifos = calloc(count + 1, sizeof(hb_fifo_t*));
 
-    // The stream needs to be open before starting the reader thead
+    // The stream needs to be open before starting the reader thread
```

Comment:
Fixing this w/o accidentally making a mess of things -- I tried to not make that mistake and planned to, I think I ended up not leaving the mistakes, but please forgive me if I did.